frontend/src/metabase/query_builder/components/MoveQuestionModal/MoveQuestionModal.tsx

```diff
@@ -48,7 +48,10 @@ export const MoveQuestionModal = ({
     boolean | undefined
   >();
 
-  const handleMove = async (destination: MoveDestination) => {
+  const handleMove = async (
+    destination: MoveDestination,
+    _deleteOldDashcards?: boolean | undefined,
```

Comment:
I think you're doing this due to variable shadowing, but this reads to me as an unused variable. Can we rename it?

Reply:
I'm so sorry, I scrolled past this and hit merge before seeing this comment. I'll get it renamed in another PR 🙏